src/standalone/Vegetation/canopy_parameterizations.jl

```diff
@@ -1099,3 +1108,459 @@ end
 A function which enforces α+τ <= 1.
 """
 enforce_albedo_constraint(α, τ) = 1 - α - τ > 0 ? α : 1 - τ
+
+
+# P-model 
+"""
+    intrinsic_quantum_yield(
+        T::FT, 
+        c::FT
+    ) where {FT}
+
+    Computes the intrinsic quantum yield of photosynthesis ϕ (mol/mol) 
+    as a function of temperature T (K) and a calibratable parameter c (unitless). 
+    The functional form is given in Bernacchi et al (2003) and is used in Stocker 
+    et al. (2020) 
+"""
+function intrinsic_quantum_yield(T::FT, c::FT) where {FT}
+    # convert to C
+    T = T - FT(273.15)
```

Comment:
If there is any chance these other constants might vary or be calibrated later, we could move them out of this and stick in the PModel struct (i.e., not hardcoded). We typically try and do this. What does @juliasloan25 think?